content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,154 @@
+
+Using proprietary protocols between the MC and an external provider device require their implementation in both the MC and the external provider device. This is a constraint imposed to the external providers who have to implement all the specific protocols of all the computer makers they want to support.
+
+Moreover, device configuration is performed in two steps as explained in this [blog post](https://developer.hpe.com/blog/storage-management-with-redfish/): 1) the MC stores the modifications in a dedicated location. 2) The modifications are uploaded to the device during next server reboot. This reboot requirement is another drawback generated by the lack of PLDM implementation.
+
+![Operational model without PLDM for RDE](/img/withoutpldmrde.png "Operational model without PLDM for RDE")
+
+As a summary, the operational model without PLDM for RDE introduces the following problems:
+
+* firmware lock-step dependency between iLO and devices
+* partial knowledge of device properties implemented iLO firmware
+* use of proprietary communication protocols between iLO and devices
+* reboot required to load new device configuration settings
+
+## Operational model with PLDM for RDE
+
+The implementation of PLDM for RDE standard, in both iLO and devices, changes the iLO role from "active" to "pass-through". In this configuration, the iLO receives HTTP requests from Redfish clients and translates them into a set of RDE requests to the device. When the PLDM dialog between the iLO and the device is over, the iLO sends back a final HTTP response to the Redfish client (See picture below).
+
+![Operational model with PLDM for RDE](/img/withpldmrde.png "Operational model with PLDM for RDE")
+
+In this model, the Management Controller and device firmware don't need to be aligned because the MC does not contain any device properties. It contains only the necessary information to translate the Redfish client requests into PLDM for RDE messages.
+
+As a result, if all the computer makers implement the PLDM for RDE protocol suite in their Redfish service, the number of communication protocols to be implemented and maintained by the device providers is reduced to one.
+
+Lastly, PLDM for RDE device modifications do not require a reboot of the server to become effective in the device. However, a reboot may be required by the operating system or by applications.
+
+## HPE implementation of PLDM for RDE
+
+On the server side, the implementation of PLDM for RDE started in iLO 5 [firmware version 2.33](https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/#changelog) and newer firmware releases implement more and more PLDM related features that can be leveraged by devices providing their support of this standard. As an example, Redfish clients connected to an iLO 5 version 2.65 can perform read and write operations on Logical Drives hosted by  [HPE SR Smart Arrays](https://www.hpe.com/psnow/doc/a00047736enw.html?jumpid=in_pdp-psnow-qs) running firmware version 5.00.
```

Comment:
Shouldn't "Logical Drives" be "logical drives"? Similar to "management controller". Since you use "LD" in other places to refer to this, don't forget to put "(LD)" immediately after this first mention.